.github/actions/label/build.js

```diff
@@ -20,8 +20,12 @@ const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
 
 /**
  *  If skipping this action.
+ *
+ * @returns {Promise<[boolean, string]>} [skip, String(check_runs)]
  **/
-const skip = async ({ core, github }) => {
+const skip = async ({ github }) => {
+  if (TITLE.includes("[depbot]")) return [true, ""]
```

Comment:
btw why don't we assert author? Anyone can open PR with [depbot] name

Reply:
hmm, ur right, we need to assert branch prefix